src/components/organisms/TriggersFormItems/TriggerSelectorSwitcher/TriggerSelectorSwitcher.tsx

```diff
@@ -0,0 +1,23 @@
+import {Segmented} from 'antd';
+
+type TriggerSelectorSwitcherProps = {
+  value: string;
+  onChange: (value: string) => void;
+};
+
+const TriggerSelectorSwitcher: React.FC<TriggerSelectorSwitcherProps> = props => {
+  const {value, onChange} = props;
+
+  return (
+    <Segmented
+      options={[
+        {label: 'BY LABEL', value: 'label'},
+        {label: 'BY NAME', value: 'name'},
+      ]}
+      value={value}
+      onChange={x => onChange(String(x))}
```

Comment:
Is there needed the casting to `String`? Looks like there needs to be some different logic then

Reply:
`Type 'number' is not assignable to type 'string'.`
I get this if I remove it. Originally yes, it has to be string or number, but I removed it as you asked

Reply:
Ok, it's fine then 👍